hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -2143,7 +2166,10 @@ def _draw(self, ir_type, kwargs, *, observe, forced, fake_forced):
                 value, kwargs=kwargs, was_forced=was_forced
             )
             size = ir_size([value])
-            if size + self.length_ir > self.max_length_ir:
+            if self.length_ir + size > self.max_length_ir:
+                debug_report(
+                    f"overrun because {self.length_ir} + {size} > {self.max_length_ir=}"
```

Comment:
```suggestion
                    f"overrun because {self.length_ir=} + {size=} > {self.max_length_ir=}"
```
?

Reply:
was semi intentional, but yeah I think with `=` is fine.